vendor/vendor.json

```diff
@@ -3925,6 +3925,12 @@
 			"revision": "d63917fcb0d53f39184485b9b6a0893af18a5668",
 			"revisionTime": "2019-07-30T05:26:13Z"
 		},
+		{
+			"checksumSHA1": "ByRdQMv2yl16W6Tp9gUW1nNmpuI=",
+			"path": "github.com/hashicorp/errwrap",
+			"revision": "8a6fb523712970c966eefc6b39ed2c5e74880354",
+			"revisionTime": "2018-08-24T00:39:10Z"
+		},
```

Comment:
Required in by `go-multierror`, I've created https://github.com/elastic/beats/issues/16737 to remove them.